test/parallel/test-fs-read-stream-inherit.js

```diff
@@ -0,0 +1,173 @@
+var common = require('../common');
```

Comment:
just copied `test-fs-read-stream.js` and wrapped all the options in `Object.create()`. is that okay?